### ISSUE TITLE
US, UK Doctors: "Change screening guidelines, lost sense of taste and smell"

### ISSUE THREAD

**Issue by codejack42** [flattenthecurve/guide#224]
**Content to be added or changed:**

**tl;dr:** _American Academy of Otolaryngology says loss of sense of taste and smell need to be added to the list of screening tools for possible COVID-19 infection. British Association of Otorhinolaryngology issues similar guidance. Loss of smell sense has been seen in patients ultimately testing positive for the coronavirus with no other symptoms. Germany reports that more than 2 in 3 confirmed cases have loss of smell sense. South Korea reports 30% of patients testing positive have had loss of smell sense as their major presenting symptom. Iran has reported a sudden increase in cases of isolated loss of smell sense. This could be very important for those caring for sick people at home, community medics, first responders, and clinicians making diagnoses, particularly when trying to remain vigilant for, identify, and isolate SBD spreaders whether asymptomatic, mildly symptomatic, or pre-symptomatic. Also potentially quite important and insightful in places like the US, where absence of testing has left dearth of data on how many people are infected but not seeking medial attention._

**Note:** Although American Academy of Otolaryngology and British Association of Otorhinolaryngology are not addressing this in view of caring for sick people at home, or in terms preventing in-home transmission, we need to assume that before long there will be many more people being cared for at home than there are being cared for in hospital, and that due to increasing self-isolation directives, the great majority of community spread will be the result of in-home transmission.

CNN Health Article "Doctors say loss of sense of smell might be Covid-19 symptom"
https://www.cnn.com/2020/03/23/health/coronavirus-symptoms-smell-intl/index.html
     points to two sources:
1) American Academy of Otolaryngology's statement, "Anosmia, Hyposmia, and Dysgeusia Symptoms of Coronavirus Disease"
https://www.entnet.org/content/aao-hns-anosmia-hyposmia-and-dysgeusia-symptoms-coronavirus-disease
    and
2) British Association of Otorhinolaryngology's statement, "Loss of sense of smell as marker of COVID-19 infection"
https://www.entuk.org/sites/default/files/files/Loss%20of%20sense%20of%20smell%20as%20marker%20of%20COVID.pdf

++++ BEGIN KEY EXCERPTS ++++
Doctors in the United States on Sunday called for the loss of sense of smell and taste to be added to the "list of screening tools" for Covid-19.
...
... symptoms of anosmia, or lack of sense of smell, and dysgeusia, or lack of taste, should be used to identify possible Covid-19 infections.
...
"Anosmia, in particular, has been seen in patients ultimately testing positive for the coronavirus with no other symptoms,"
...
Those symptoms "warrant serious consideration for self-isolation and testing of these individuals,"
...
"... good evidence from South Korea, China and Italy that significant numbers of patients with proven COVID-19 infection have developed anosmia,"
...
"In Germany it is reported that more than 2 in 3 confirmed cases have anosmia. In South Korea, where testing has been more widespread, 30% of patients testing positive have had anosmia as their major presenting symptom in otherwise mild cases."
...
"We propose that these symptoms be added to the list of screening tools for possible COVID-19 infection."
...
"Anosmia, hyposmia, and dysgeusia ... should alert physicians to the possibility of COVID-19 infection and warrant serious consideration for self-isolation and testing of these individuals."
...
... there have been a rapidly growing number of reports of a significant increase in the number of patients presenting with anosmia in the absence of other symptoms –this has been widely  shared  on medical  discussion  boards  by  surgeons  from  all regions  managing  a high incidence of cases. Iran has reported a sudden increase in cases of isolated anosmia, and many  colleagues from the  US,  France and  Northern Italy have  the  same experience.
...
... these patients may be some of the hitherto hidden carriers that have facilitated the rapid spread ofCOVID-19.  Unfortunately,  these  patients  do  not  meet  current  criteria  for  testing  or  self-isolation.
...
... could  potentially  be  used  as  a  screening  tool  to  help  identify otherwise asymptomatic patients, who could then be better instructed on self-isolation.
...
There is potential that if any adult with anosmia but no other symptoms was asked to self-isolate for seven days, in addition to the current symptom criteria used to trigger quarantine, we might be able to reduce the number of otherwise asymptomatic individuals who continue to act as vectors, not realising the need to self-isolate.
...
It will also be an important trigger for healthcare  personnel  to  employ  full  PPE  and  help  to  counter  the  higher  rates  of  infection found amongst ENT surgeons compared to other healthcare workers.
++++ END KEY EXCERPTS ++++
